frontend/src/metabase/components/DebouncedFrame.jsx

```diff
@@ -32,12 +32,18 @@ export default class DebouncedFrame extends React.Component {
     };
   }
 
-  setSize = (width, height) => {
+  updateSize = () => {
     this._transition = false;
-    this.setState({ width, height }, this._updateTransitionStyle);
+
+    const { width, height } = this.props;
+    if (width !== this.state.width || height !== this.state.height) {
+      this.setState({ width, height }, this._updateTransitionStyle);
+    } else {
+      this._updateTransitionStyle();
```

Comment:
_updateTransitionStyle needs to be always be called even if `setState` isn't called otherwise, the UI will stick to a loading state.